tools/ktx/basis_utils.h

```diff
@@ -144,49 +144,13 @@ enum class EncodeCodec {
             deterministic).</dd>
     </dl>
 
-    <dt>
-        common:
-    </dt>
-    <dd>
-        Common options.
-    </dd>
+    @snippet{doc} ktx/encode_utils_common.h command options_codec_common
```

Comment:
I think this should be options_encode_common. I think we should use codec only for the BasisLZ, UASTC choice.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/817/commits/36912226d37fcd790fd2ae11d97ddeafe8ab7e9e and  https://github.com/KhronosGroup/KTX-Software/pull/817/commits/8c6cb1ddebd6988735182ccafc43cdb3426a7fd9